circle.yml

```diff
@@ -25,11 +25,11 @@ test:
     # 0) runs unit tests w/ H2 local DB. Runs against H2, Mongo, MySQL
     # 1) runs unit tests w/ Postgres local DB. Runs against H2, SQL Server
     # 2) runs unit tests w/ MySQL local DB. Runs against H2, Postgres, SQLite
-    # 3) runs Eastwood linter
-    # 4) Bikeshed linter
+    # 3) runs unit tests w/ H2 local DB. Runs against H2, Redshift
```

Comment:
That didn't have anything to do with the databases though. That was just CircleCI being horribly unreliable and bad at life (which at this point shouldn't surprise anybody). Looks like `sudo apt-get update` died after stalling for ~22 minutes